debian/vyos-1x.postinst

```diff
@@ -109,11 +109,10 @@ if ! grep -q '^hostsd' /etc/group; then
     addgroup --quiet --system hostsd
 fi
 
-# add dhcpd user for dhcp-server
-if ! grep -q '^dhcpd' /etc/passwd; then
-    adduser --quiet --system --disabled-login --no-create-home --home /run/dhcp-server dhcpd
-    adduser --quiet dhcpd hostsd
-fi
+# Add _kea user for kea-dhcp{4,6}-server to vyattacfg
+# Note that kea-common should have been installed as dependency and the user
+# '_kea' should exist
+adduser --quiet _kea vyattacfg
```

Comment:
Please add a test if user _kea exists.

```bash
if  grep -q '^_kea' /etc/passwd; then
    adduser --quiet _kea vyattacfg
fi
```

Reply:
If the user exists, should it use `usermod` to still add the group?

Reply:
`adduser` is fine doing this

```
cpo@LR1.wue3# sudo adduser _kea vyattacfg
Adding user `_kea' to group `vyattacfg' ...
Done.
```